include/clang/CrossTU/CrossTranslationUnit.h

```diff
@@ -159,6 +161,7 @@ class CrossTranslationUnitContext {
   llvm::SmallPtrSet<const FunctionDecl *, 8> InvalidFunctions;
   CompilerInstance &CI;
   ASTContext &Context;
+  std::unique_ptr<ASTImporterLookupTable> LookupTable;
```

Comment:
I think we could have a regular field here (unless we want to create this lazily.)